clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -511,4 +511,10 @@ public void recordNearlineLocalBrokerToReadyToServeLatency(double value, long cu
     nearlineLocalBrokerToReadyToServeLatencySensor.record(value, currentTimeMs);
   }
 
+  public static double unAvailableToZero(double value) {
+    /* When data is unavailable, return 0 instead of NaN or Infinity. Some metrics are initialized to -INF.
+     This can cause problems when metrics are aggregated. Use only when zero makes semantic sense.
+    */
```

Comment:
What if we initialize these metrics to 0? Do you think we would still see infinite in this case?